pkg/apis/projectcalico/v3/ippool.go

```diff
@@ -65,6 +65,9 @@ type IPPoolSpec struct {
 	// When disabled is true, Calico IPAM will not assign addresses from this pool.
 	Disabled bool `json:"disabled,omitempty"`
 
+	// Disable exporting routes from this IP Pool’s CIDR over BGP. [Default: false]
+	DisableBGPExport bool `json:"disableBGPExport,omitempty" validate:"omitempty"`
```

Comment:
Does this need to be a pointer so that it works with pre-existing IPPools that do not have this field?

Reply:
I was under the impression that since we have `omitempty`, and the default value is false, we wouldn't need it to be a pointer, but I'll investigate some more to be sure. Thanks @mgleung!

Reply:
If we make it a pointer, we'd default it to false anyway, so would effectively just be writing code to implement what golang does by default. We'd need to write defaulting code on write and on read for existing IP pools. That's something we've done before, and isn't too hard, but I'm not sure what making this a pointer would achieve other than more closely matching other optional fields (which, is not something we should do lightly)